torch_em/segmentation.py

```diff
@@ -250,8 +250,11 @@ def default_segmentation_dataset(
     is_seg_dataset=None,
     with_channels=False,
     with_label_channels=False,
+    verify_paths=True,
 ):
-    check_paths(raw_paths, label_paths)
+    if verify_paths:
+        check_paths(raw_paths, label_paths)
```

Comment:
@constantinpape Apparently this was the most expensive part (the one in `ImageCollectionDatasets` costs almost nothing). With this, the entire dataset can be loaded in ~30 seconds! ;)

Thanks for the suggestion!

Reply:
We should probably expose this in `default_segmentation_loader` / `default_segmentation_dataset` too.